src/audio_unit/mod.rs

```diff
@@ -1,21 +1,20 @@
 mod delay;
+mod gain;
```

Comment:
I read the modules chapter of the book - this still doesn't seem 100% clear, you have to declare the existence of this other module via `mod` but then use `use` to (unprefixed) reference its public members? Or if you omit the `use` can you still reference `gain::whatever` because of just the `mod gain;`?

Reply:
> Or if you omit the use can you still reference gain::whatever because of just the mod gain;?

Yeah, that's right (I think).

- without the `mod`, nothing can use anything from `gain`
- with `mod`, anything in `audio_unit` can refer to the contents of `gain` as `gain::Gain`, etc
- with `use gain::Gain`, anything in `audio_unit` can use `Gain` without namespacing
- with `pub use gain::Gain`, `Gain` is re-exported, so anything that imports `audio_unit` can use it directly (`audio_unit::Gain`, or `use crate::audio_unit::Gain` followed by the use)